src/components/HeroPicture.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+
+function HeroPicture() {
+  return (
+    <div className="hero-picture">
+    
+    <img src="./statue.png" alt="Generic problematic statue"></img>
+   
+    </div>
+  );
+}
```

Comment:
Why a component just to render an img? This probably should be in one component with the text in the Hero